installer/lib/phoenix_new.ex

```diff
@@ -153,7 +164,8 @@ defmodule Mix.Tasks.Phoenix.New do
   def run(app, mod, path, opts) do
     db = Keyword.get(opts, :database, "postgres")
     ecto = Keyword.get(opts, :ecto, true)
-    brunch = Keyword.get(opts, :brunch, true)
+    html = Keyword.get(opts, :html, true)
+    brunch = Keyword.get(opts, :brunch, html && true)
```

Comment:
Let's keep the `--api` flag out for now, we can start another discussion for it. :)